### PR TITLE
feat: add profile update

### PR DESCRIPTION
**Pull request by janniks**
🚧 fork

### CODE REVIEW

File: packages/connect/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "@stacks/prettier-config": "0.0.8",
     "@stacks/transactions": "^5.0.0",
     "jsontokens": "^4.0.1",
+    "@stacks/profile": "^5.0.0",
```
Comment (friedger) [inline packages/connect/package.json:34]: Use "5.0.3-pr.ea8bc25.0" here, please :-)

**Comment by janniks** [comment]
latest alpha of this branch: `5.0.6-alpha.5071c1c.0`

**Comment by janniks** [comment]
Changes look good 👍
Fixed the CI issues.

Do you think these types of feature-additions would be easier to contribute to if connect and stacks.js would be in the same repo?

**Comment by janniks** [comment]
@friedger `5.0.6-alpha.dd8ecff.0` is the latest using https://github.com/hirosystems/connect/pull/279

**Pull request merged**